sdk-python/awaazde/utils.py

```diff
@@ -0,0 +1,137 @@
+import csv
+from datetime import datetime
+
+from constants import CommonConstants
+
+from . import AwaazDeAPI
+
+
+class ADMessageUtils(object):
```

Comment:
I feel most of the functionality in this util is generic and/or can be subclassed with a very light object-specific util. The boundary between this util and the methods that rely on it is very fuzzy. I am not seeing a clean seperation/design.

Can we take a step back and work on your design in a doc? We don't need to see the implementation details but the utils, models, and commands with clearly named functions at least spec'd out will be helpful.